pkg/ccl/backupccl/testdata/show_backup_validate/show_backup

```diff
@@ -0,0 +1,15 @@
+validate-backup invalidDependOn_21.1
```

Comment:
the error message seems to suggest it's a  `invalidDependedOnBy` case, so should we rename it to `invalidDependedOnBy_21.1`?